cider-velox/src/substrait/VeloxPlanFragmentToSubstraitPlan.cpp

```diff
@@ -103,6 +103,31 @@ void VeloxPlanFragmentToSubstraitPlan::reconstructVeloxPlan(
         return std::make_shared<FilterNode>(
             filterNode->id(), filterNode->filter(), planBuilder_->planNode());
       });
+    } else if (auto orderByNode = std::dynamic_pointer_cast<const OrderByNode>(*riter)) {
+      planBuilder_->addNode([&](std::string id, core::PlanNodePtr input) {
```

Comment:
seems `id` and `input` not used?
so why the lambda function should pass this two args?

Reply:
See here.
```
  /// Add a user-defined PlanNode as the root of the plan. 'func' takes
  /// the current root of the plan and returns the new root.
  PlanBuilder& addNode(
      std::function<core::PlanNodePtr(std::string nodeId, core::PlanNodePtr)>
          func) {
    planNode_ = func(nextPlanNodeId(), planNode_);
    return *this;
  }
```